nexus/reconfigurator/execution/src/datasets.rs

```diff
@@ -70,19 +69,7 @@ pub(crate) async fn ensure_crucible_dataset_records_exist(
 
         // Map progenitor client strings into the types we need. We never
         // expect these to fail.
```

Comment:
This comment can go away too I think? It doesn't look like we're doing any other progenitor client type conversion after this change.

Reply:
Done